rust/crd/src/catalog/delta_lake.rs

```diff
@@ -0,0 +1,11 @@
+use super::hive::HiveConnector;
+use serde::{Deserialize, Serialize};
+use stackable_operator::schemars::{self, JsonSchema};
+
+#[derive(Clone, Debug, Deserialize, Eq, JsonSchema, PartialEq, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct DeltaLakeConnector {
+    /// The Hive connector exposes the same config
+    #[serde(flatten)]
+    pub hive: HiveConnector,
+}
```

Comment:
Not sure i like this. Why not copy paste this for now, in case anything changes in the future? If not i think to add to the confusion i would call the variable `delta_lake` instead of `hive`?
```suggestion
    /// The Hive connector exposes the same config
    #[serde(flatten)]
    pub hive: HiveConnector,
```

Reply:
I think this is a trade-off between us doing changes to hive and missing it in other connectors vs. doing a change in hive that can not be added to the others.
I have no strong opinions, so let's stick to copy/paste

Reply:
I dont know much about the data lake connector, i leave the decision up to you.

Reply:
Already changed it back to copy/paste and added a comment so everything should be fine :)